topology.graphqls

```diff
@@ -36,6 +42,26 @@ type Node {
     isReal: Boolean!
 }
 
+# Node in ServiceInstanceTopology
+type ServiceInstanceNode {
+    # The global id of each node,
+    # 1. Service id
+    serviceId: ID!
+    # The literal name of the #serviceId.
+    serviceName: String!
+    # The global id of each node,
+    # 1. Instance id
+    instanceId: ID!
```

Comment:
This should be renamed as id, like `Node#id`. Becuase this is used on `Call` object.